UGENE-6996. Add Russian translation for some context menu for MSA editor

yalgaer opened the pull request.

yalgaer committed:
UGENE-6996. Add Russian translation for some context menu for MSA ediror

DmitriiSukhomlinov reviewed (dismissed):
Approved, but what about the other unfinished translations?

evelinabis commented on src/corelibs/U2View/transl/russian.ts:
```diff
@@ -2259,7 +2259,7 @@ Double-click to collapse the branch</source>
         <location filename="../src/ov_sequence/ADVClipboard.cpp" line="280"/>
         <source>Copy annotation</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать анноттацию</translation>
```
Typo

evelinabis commented on src/corelibs/U2View/transl/russian.ts:
```diff
@@ -4570,19 +4570,19 @@ Please, check external tools in the settings.</source>
+        <location filename="../src/ov_msa/MSAEditorSequenceArea.cpp" line="901"/>
         <source>Export highlighting finished successfully</source>
-        <translation type="unfinished">Export highligtning finished successfully</translation>
+        <translation>Export highligtning finished successfully</translation>
```
Why is the "unfinished" removed here?

yalgaer committed:
Add missed translations for new texts in v38

yalgaer commented:
> Approved, but what about the other unfinished translations?

There are 2 kind of unfinished translations:
1. Legacy: from the old releases. These will be fixed later (I do not have any concrete date)
2. New texts: introduced in this release. These should be fixed, so we have no untranslated texts. 

I reviewed the patch again and fixed all 2.
